hatch/src/cli/commands/new.rs

```diff
@@ -69,30 +69,25 @@ impl<'command> Command<'command> for New {
   }
 
   fn execute(&self, args: &ArgMatches<'command>) -> Result<Vec<Project>, HatchError> {
-    match fs::create_dir(self.project_path(args) + self.project_name(args).unwrap().as_str()) {
+    let name = self.project_name(args).unwrap();
+
+    match fs::create_dir(self.project_path(args) + &name.as_str()) {
```

Comment:
How is this reducing string allocations? I still see you gluing two strings together.